src/scalar.jl

```diff
@@ -173,21 +311,21 @@ as(::Type{Real}, left, right) =
 
 as(::Type{Real}, ::Infinity{false}, ::Infinity{true}) = Identity()
 
-as(::Type{Real}, left::Real, ::Infinity{true}) = ShiftedExp(true, left)
+as(::Type{Real}, left::Real, ::Infinity{true}) = TVShift(left) ∘ TVExp()
 
-as(::Type{Real}, ::Infinity{false}, right::Real) = ShiftedExp(false, right)
+as(::Type{Real}, ::Infinity{false}, right::Real) = TVShift(right) ∘ TVNeg() ∘ TVExp()
 
 function as(::Type{Real}, left::Real, right::Real)
     @argcheck left < right "the interval ($(left), $(right)) is empty"
-    ScaledShiftedLogistic(right - left, left)
+    TVShift(left) ∘ TVScale(right - left) ∘ TVLogistic()
```

Comment:
IMO it would be good to promote `left` and `right` to a common type here - both to reduce the amount of different types (and hence compilation etc.) and to avoid inaccuracies due to less accurate calculations in one part of the chain (e.g., one could think of something extreme like `left::BigFloat` + `right::Int` or less extreme `left::Float32` and `right::Float64`.
```suggestion
    shift, scale = promote(left, right - left)
    TVShift(shift) ∘ TVScale(scale) ∘ TVLogistic()
```